### ISSUE TITLE
[Bug] [a11y] Cannot type text in the search bar on some tablets

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#3583]
## Steps to reproduce
1. Turn Talkback or Voice Assistant on.
1. Tap on the search bar and type some text.

### Expected behavior
The text is displayed in the search bar.

### Actual behavior
Nothing is displayed in the search bar, even though the assistant reads the letters that are typed.
Works as expected on phones.

### Device information
* Android device: Samsung Galaxy Tab S3 (Android 8), Sony Xperia Z2 (Android 6.0.1)
* Fenix version: 1.0.0-rc.2 (#11700147)

Not reproducible on Huawei MediaPad M3 Lite 10 (Android 7.0)

**Comment by softvision-miralobontiu** [comment]
Workaround: tap on a letter, then double tap that letter and it is displayed. Or, is this expected?

**Comment by mcarare** [comment]
@softvision-miralobontiu Did you you use Voice Assistant (not Talkback) in all tests? By default, Voice Assistant types letters from keyboard by double tap, bot single tap. There is also a setting to enable QuickTyping (or "Rapid key input" in some versions). If, so, the behaviour described is as expected.

**Comment by softvision-miralobontiu** [comment]
I used Voice Assistant.

**Comment by mcarare** [comment]
Then I think this can be closed, as it is the expected behaviour of Voice Assistant.
